### DISCUSSION TITLE
Option for preferred pricing for trade exchange searches

### DISCUSSION THREAD

**Discussion by SoapGOD** [SnosMe/awakened-poe-trade#1262] [Ideas]
When you are price checking scarabs, essence, or other currencies it often defaults to checking the divine orb value for those items with bulk. When you don't have enough to sell for divine orbs or you prefer to just price in chaos you end up having to swap from divine to chaos per item causing each one to count at 2 searches and then you hit your limit of "x/10 in 90 seconds" after only 5 searches. 

Was wondering if there could be an option to always check chaos or divine by default, so if you prefer chaos or don't have much stock then you can just switch to chaos and not get search limited so quickly.

**Comment by SnosMe** [comment]
#1225

**Comment by SoapGOD** [comment]
So in the other post you mention the filter being this way is so when bulk trading people wont sell items for the wrong price?

Could there be an advanced option to ignore that option. 

I might be doing something wrong but the "Remember buyout currency filter" doesn't do anything when checked or not checked, so not sure what it is referring to.

**Reply by SnosMe** [reply]
that issue is not closed, which means i'll implement it for bulk trade some time

**Reply by SoapGOD** [reply]
ahh ok thank you, and thanks for the tool.
